projects/_posts/2015-12-01-mmmm.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: project
+title: My Mind & My Money
+handle: MMMM
+image: /assets/images/projects/MMMM/MMMM_logo.png
+contributors: [msutherland, alaird, rpoudel]
+grant:
+grant_url:
+category: projects
+tags: [fMRI]
```

Comment:
@ranjita Sorry about this but could you change this (`tags: [fMRI]`) to `tags: [fmri]`? Just commit the change normally and the PR will update automatically.